RELEASE-NOTES.md

```diff
@@ -1,5 +1,17 @@
 # Prefect Release Notes
 
+## Release 2.19.7
+
+### Fixes
+- Fix bug where assignments inside functions are evaluated when running `prefect deploy` - https://github.com/PrefectHQ/prefect/pull/14403
+
```

Comment:
should we also add this PR, even though it's maintenance, as it was requested by a user?
https://github.com/PrefectHQ/prefect/pull/14370